src/components/DotIndicatorMessage.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import styles from '../styles/styles';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import colors from '../styles/colors';
+import variables from '../styles/variables';
+import Text from './Text';
+
+const propTypes = {
+    /** The messages to display  */
+    // eslint-disable-next-line react/forbid-prop-types
+    messages: PropTypes.object,
+
+    /** The type of message, 'error' shows a red dot, 'success' shows a green dot */
+    type: PropTypes.oneOf(['error', 'success']).isRequired,
+
+    /** Additional styles to apply to the container */
+    // eslint-disable-next-line react/forbid-prop-types
+    style: PropTypes.arrayOf(PropTypes.object),
+};
+
+const defaultProps = {
+    messages: null,
```

Comment:
assuming we can make `messages` an Array prop type then we'd want
```suggestion
    messages: [],
```